app/scripts/metamask-controller.test.js

```diff
@@ -952,7 +952,7 @@ describe('MetaMaskController', () => {
       it('should call keyringController.getAccounts', async () => {
         expect(
           metamaskController.keyringController.getAccounts,
-        ).toHaveBeenCalledTimes(2);
+        ).toHaveBeenCalledTimes(3);
```

Comment:
Resolves error in jest test `MetaMaskController > MetaMaskController Behaviour > unlockHardwareWalletAccount > should call keyringController.getAccounts`.

Likely due to changes introduced by the keyring-controller major version update, specifically in this ticket https://github.com/MetaMask/core/issues/3699 and its dependencies.